profiles/utilities.py

```diff
@@ -15,3 +17,10 @@ def generate_random_string(length: int, chars: str = string.ascii_letters + stri
 
 def remove_none_values(items: dict) -> dict:
     return dict(filter(lambda item: item[1] is not None, items.items()))
+
+
+def string_to_name(name: str) -> Name:
```

Comment:
I understand this is temporary

Reply:
Only if we want to consider successful auth then immediate API failure as a Profiles auth failure. (Otherwise we only have access to a single-string name.)